in_text/text_04_predictions.yml

```
@@ -164,23 +164,23 @@ entities:
       attr-defs: this data release
       data-min: NA
       data-max: NA
-      data-units: integer
+      data-units: NA
     -
       attr-label: issue_time
       attr-def: >-
         forecast issue time
       attr-defs: this data release
       data-min: NA
       data-max: NA
-      data-units: days since {forecast issue time}
+      data-units: days since {forecast issue time} 
```

Did the time and issue_time variables get mixed up? I see actual dates in issue_time and integers in time. That makes me think this should be the unit for time, not issue_time
```suggestion
      data-units: days since {forecast issue time}
```

---

THis was pulled from last years data release and this attribute was documented in this way (see [repo](https://github.com/USGS-R/drb-temp-data-release/blob/main/in_text/text_05_predictions.yml))

---

That unit description is correct. It will be days since the first forecast issue time of the netCDF file which is stored in the metadata of the netCDF file.